.env.example

```diff
@@ -0,0 +1,3 @@
+AWS_ACCESS_KEY_ID=
```

Comment:
In which cases will we be using a .env, since you created the `configuration.yaml` file?

Reply:
Env vars have priority against config file. This `.env` is used by docker. Just for containers for now.

Reply:
Now we have just `.env` file